src/main/java/Entities/Property.java

```diff
@@ -3,60 +3,53 @@
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.util.ArrayList;
+import java.util.Objects;
 import java.util.Scanner;
 
 public class Property extends Cell {
-    private Player ownedBy;
-    private int houses;
-    private int rent;
     private String name;
     private String colour;
+    private int cost;
+    private int rent;
+    private int rent1H;
+    private int rent2H;
+    private int rent3H;
+    private int rent4H;
+    private int rentHotel;
+    private Player ownedBy;
     private int mortgageValue;
     private int houseCost;
-    private int cost;
+    private int houses;
     private boolean mortgaged;
 
-    public Property (String name, String colour, int cost, int rent, int mortgageValue, int houseCost) {
-        this.ownedBy = null;
-        this.houses = 0;
-        this.cost = cost;
+    public Property (String name, String colour, int cost, int houseCost, int rent,
+                     int rent1H, int rent2H, int rent3H, int rent4H, int rentHotel,
+                     Player owner, int mortgageValue, int houses, boolean mortgaged) {
```

Comment:
Honestly this is just me being picky, but having a constructor with such a large amount of arguments seems bizarre. Couldn't we reduce the amount of arguments by grouping all the rent values together? The code is likely fine either way, but I'm wondering if this might be a code smell.

Reply:
I think this is also a code smell, instead can't you just pass in the data array instead?